cmd/srrs/main.go

```diff
@@ -174,10 +177,29 @@ func main() {
 			}
 		}()
 
+		// https server
 		tlsErrCh := make(chan error, 1)
-		go func() {
-			// TODO: Start tlsServ
-		}()
+		cert := *certPath
+		key := *keyPath
+		if cert != "" && key != "" {
+			tlsLogger := logger.With(zap.String("listen_addr_tcp", *tlsAddr))
+			tlsSrv := &http.Server{
+				Addr:     *tlsAddr,
+				ErrorLog: zap.NewStdLog(tlsLogger),
+				Handler:  mux,
+			}
+
+			go func() {
+				tlsLogger.With(zap.String("certificate path", cert), zap.String("key path", key))
```

Comment:
this does not do anything. you can simply specify the fields on the line below (i.e. `tlsLogger.Info("...", zap.String(..., ...), zap.String(..., ...))`)